Makefile

```diff
@@ -85,10 +84,6 @@ unit: ## Run tests
 .PHONY: test  ## Run formatter, linter and tests
 test: generate fmt lint unit
 
-.PHONY: test-integration
-test-integration: ## Run integration tests
-	source ./hack/fetch_ext_bins.sh; fetch_tools; setup_envs; go test -v -tags=integration ./test/integration/...
-
 .PHONY: test-e2e
 test-e2e: ## Run e2e tests
```

Comment:
unsure about the idea here, this is currently broken as well.

Reply:
Looks like copy paste. We can remove it.